### DISCUSSION TITLE
Other tool options

### DISCUSSION THREAD

**Discussion by mjtarr** [slotThe/RemarkableLamyEraser#15] [Ideas]
Beyond the erase option there is currently only select:

"select: Changes to select tool. On deactivation, changes back to writing utensil"

Any interest from users in creating a new tool that is highlight - ideally so that it toggles between the writing pen and the highlight pen? I end up swapping a good bit while marking up PDFs...

-mike

**Comment by slotThe** [comment]
You're looking for something like https://github.com/slotThe/RemarkableLamyEraser/pull/12 right?

**Comment by mjtarr** [comment]
Exactly. Sorry, I hadn’t seen #12
